script/release.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * This is a helper script to tag and push a new release. GitHub Actions use
+ * release tags to allow users to select a specific version of the action to use.
+ *
+ * See: https://github.com/actions/typescript-action#publishing-a-new-release
+ *
+ * This script will do the following:
+ *
+ * 1. Get the latest release tag
+ * 2. Prompt the user for a new release tag
+ * 3. Tag the new release
+ * 4. Push the new tag to the remote
+ *
+ * Usage: node script/release.js
+ */
+
+import readline from 'node:readline/promises'
+import { stdin as input, stdout as output } from 'node:process'
+import { simpleGit } from 'simple-git'
+
+// Terminal colors
+const OFF = '\x1B[0m'
+const RED = '\x1B[0;31m'
+const GREEN = '\x1B[0;32m'
+const BLUE = '\x1B[0;34m'
+
+const git = simpleGit()
+
+try {
+  // Get the latest release tag
+  let latestTag = (await git.tags()).latest
```

Comment:
This is a little different than the original bash command - `git describe --tags --match='v*.*.*' "$(git rev-list --tags --max-count=1)"`. I guess non-version tags might cause issues. But it's probably an edge case, and the latest is only informative anyway, so no blocker.